src/vaadin-dialog.html

```diff
@@ -289,6 +391,172 @@
             e.preventDefault();
           }
         }
+
+        _addDraggableEventListeners() {
+          const overlay = this.$.overlay.$.overlay;
+          const originalBounds = {top: 0, left: 0};
+          const originalMouse = {top: 0, left: 0};
+          const stopDrag = () => {
+            window.removeEventListener('mouseup', stopDrag);
+            window.removeEventListener('mousemove', drag);
+          };
+          const drag = (e) => {
+            this._snapToPixels();
+            const top = originalBounds.top + (e.pageY - originalMouse.top);
+            const left = originalBounds.left + (e.pageX - originalMouse.left);
+            this._setBounds({top, left});
+          };
+          const startDrag = (e) => {
+            if (this.draggable) {
+              const target = e.target;
+              const isOverlayPart = target.matches ? target.matches('[part="overlay"]') : target.msMatchesSelector('[part="overlay"]');
+              const isContentPart = target.matches ? target.matches('[part="content"]') : target.msMatchesSelector('[part="content"]');
+              if (isOverlayPart || isContentPart) {
+                e.preventDefault();
+                if (this.modeless) {
+                  this._moveOverlayToTop();
```

Comment:
This among some other lines of code I was able to just comment out but still get a passing build = untested functionality.

Reply:
Should be tested now.

Reply:
Yes this one is, but there is still untested code in the PR, like the cleanup of the listeners (what stopResize does for example) and the other invocation to `_moveOverlayToTop`

Reply:
Added tests for the cleanup of the listeners and all calls of `_moveOverlayToTop`.